trusted_shuffler/server/src/http.rs

```diff
@@ -32,8 +32,11 @@ struct HttpRequestHandler {
 
 #[async_trait]
 impl RequestHandler for HttpRequestHandler {
-    async fn handle(&self, request: PlaintextRequest) -> anyhow::Result<PlaintextResponse> {
-        let mut request = trusted_shuffler_to_hyper_request(request)?;
+    async fn handle(
+        &self,
+        plaintext_request: PlaintextRequest,
+    ) -> anyhow::Result<PlaintextResponse> {
+        let mut request = HyperRequestWrapper::build(plaintext_request)?.0;
```

Comment:
This feeld a bit weird (always returning `wrapper.0` immediately after calling `build`). If the output of `build` is never used as a wrapper, perhaps the output type should change? Same applies to `HyperResponseWrapper::build`

Reply:
I used this Wrapper to `impl` methods for a hyper request, i.e., a `Request<Body>`, here. I think I cannot get around this, but let me know if I am missing something!

Reply:
I think this is fine. Another way to  achieve it without the wrapper is to define a trait and implement the trait directly on the hyper request and response structs. A downside to traits is that you have different signatures (e.g. async vs not async) so would need two very similar traits (which might be a bit confusing).

I would probably have approached it somewhat differently by implementing these functions with different names on `PlaintextRequest` and `EncryptedRequest`. If there wasn't a need for async it could have been done by implementing something like `TryFrom` and `TryInto`. Having said that, I think it is more of a question of personal preference, so feel free to go ahead with this approach.